crates/bevy_gizmos/src/arrows.rs

```diff
@@ -58,11 +65,22 @@ impl<T: GizmoConfigGroup> Drop for ArrowBuilder<'_, '_, '_, T> {
         // - extend the vectors so their length is `tip_length`
         // - rotate the world so +x is facing in the same direction as the arrow
         // - translate over to the tip of the arrow
-        let tips = tips.map(|v| rotation * (v.normalize() * self.tip_length) + self.end);
-        for v in tips {
+        let tips_end = tips.map(|v| rotation_end * (v.normalize() * self.tip_length) + self.end);
+        for v in tips_end {
             // then actually draw the tips
             self.gizmos.line(self.end, v, self.color);
         }
+        // If this arrow is double ended, we draw the other head at the start point
```

Comment:
```suggestion
```
IMO this comment is not needed, since the code is pretty clear on what it is doing.